src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
```

Comment:
The `href` attribute in the `<link>` tag is pointing to a SCSS file (`styles/index.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS file into a CSS file and link to the resulting CSS file instead.